internals/overlord/checkstate/manager.go

```diff
@@ -418,16 +411,17 @@ type refreshInfo struct {
 
 // checkData holds the metrics and other data for a single check.
 type checkData struct {
-	name         string
-	level        plan.CheckLevel
-	startup      plan.CheckStartup
-	status       CheckStatus
-	failures     int
-	threshold    int
-	changeID     string
-	successCount int64
-	failureCount int64
-	refresh      chan refreshInfo
+	name          string
+	level         plan.CheckLevel
+	startup       plan.CheckStartup
+	status        CheckStatus
+	successes     int
+	failures      int
+	threshold     int
+	changeID      string
+	successMetric int64
+	failureMetric int64
```

Comment:
I renamed these fields to avoid confusion with `successes` and `failures`. These are always-increasing numbers only used for the Prometheus metrics.